api/nakadi-event-bus-api.yaml

```diff
@@ -59,21 +347,42 @@ paths:
             items:
               $ref: '#/definitions/Topic'
         '401':
-          description: User is not authenticated
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
           schema:
             $ref: '#/definitions/Problem'
+
         '503':
           description: Not available
           schema:
             $ref: '#/definitions/Problem'
+
   '/topics/{topic}/events':
     get:
       tags:
-        - low-level-api
-      summary: |
+        - topic-api
+        - draft
+      description: |
         Starts a stream delivery for the specified partitions of the given topic.
-        The tracking of the current position in the partitions and of which
-        partitions is being read is in the responsibility of the client.
+
+        The event stream is formated as a sequence of `EventStreamBatch`es separated by `\n`. Each
```

Comment:
Well, there's \n\n between "logical batches" (I don't actually know if it is good or bad)